tests/python_client/testcases/test_search.py

```diff
@@ -1788,6 +1788,7 @@ def test_search_binary_substructure_flat_index(self, auto_id, _async, index, is_
             res.done()
             res = res.result()
         assert res[0].distances[0] == 0.0
+        assert len(res) <= default_limit
```

Comment:
The "check task" should be deleted in the former search request.

Reply:
> The "check task" should be deleted in the former search request.

Done.